### DISCUSSION TITLE
Original + Translated

### DISCUSSION THREAD

**Discussion by Jyaris** [SubtitleEdit/subtitleedit#5223] [Q&A]
Hi there, 

Is there a way to have two different timing control for the original text and the translated one when working on both at the same time? 
Since sentences are sometimes longer in a language than the other, timing can also be different, but it's quicker to work on both directly at the same time.

Also, is there a way to still have this button to split one subtitle in two lines? It disappears when working on both original and translated.

Thanks!

**Comment by LeonCheung** [comment]
For the first point, maybe you mean something like this: #1344
For the second point, I think that's because they are not always meet the "Split line" rule together due to the different lengths they may have. However, I think an ideal way to do it manually would be like this: #3030
